consul/backup.sls

```diff
@@ -8,20 +8,20 @@
 {%- set s3_bucket   = salt['pillar.get']('consul_backup:s3:bucket') %}
 {%- set s3_path     = salt['pillar.get']('consul_backup:s3:path', 'consul') %}
 {%- set s3_profile  = salt['pillar.get']('consul_backup:s3:profile', 'default') %}
-{%- set user        = salt['pillar.get']('consul_backup:user', 'consul') %}
+{%- set user        = salt['pillar.get']('consul_backup:user', 'root') %}
```

Comment:
This is suspicious, is this solely to write to `/etc`, or was there another issue this addressed?